### PR TITLE
Update PayButtonEnable.cshtml

### PR DESCRIPTION
**Pull request by sgracia13**
Edit PayButton Title

### CODE REVIEW

**Comment by britttttk** [comment]
closes #1940

**Review by dennisreimann — APPROVED** [review]
Works, thank you! 🍰 

after

**Comment by pavlenex** [comment]
Hey @sgracia13 congrats on your first PR. 🎉 

Was just testing this and on my end it looks like this when outside invoices aren't allowed. 
Screenshot 2020-10-07 at 10 09 10
`Pay Button: PayButton` in H2.

However when that button is clicked as Dennis mentioned then it's good.
Screenshot 2020-10-07 at 10 11 20

Can you please change the H2 tile of the above screenshot to make it consistent?

File: BTCPayServer/Views/Stores/PayButtonEnable.cshtml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿@{
     Layout = "../Shared/_NavLayout.cshtml";
-    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton, "Please confirm you want to allow outside world to create invoices in your store (via API).");
+    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton);
     ViewBag.MainTitle = "Pay Button";
```
Comment (dennisreimann) [inline BTCPayServer/Views/Stores/PayButtonEnable.cshtml:4]:
```suggestion
```

@sgracia13 You can delete this line, then we are good to go 👍 

pay

File: BTCPayServer/Views/Stores/PayButtonEnable.cshtml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿@{
     Layout = "../Shared/_NavLayout.cshtml";
-    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton, "Please confirm you want to allow outside world to create invoices in your store (via API).");
+    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton);
```
Comment (mariodian) [inline BTCPayServer/Views/Stores/PayButtonEnable.cshtml:3]: I think this should be: `ViewData.SetActivePageAndTitle(StoreNavPages.PayButton, "Outside Access");` or something along the lines to make it clear what page you're on.

Comment (Kukks) [reply BTCPayServer/Views/Stores/PayButtonEnable.cshtml:3]: the title setter below overrides it

File: BTCPayServer/Views/Stores/PayButtonEnable.cshtml (hunk 1 above)
Comment (Kukks) [reply BTCPayServer/Views/Stores/PayButtonEnable.cshtml:4]: should be part of `SetActivePageAndTitle`

**Review by Kukks — CHANGES_REQUESTED** [review]

**Review by Kukks — APPROVED** [review]

**Review by bolatovumar — APPROVED** [review]

File: BTCPayServer/Views/Stores/PayButtonEnable.cshtml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿@{
     Layout = "../Shared/_NavLayout.cshtml";
-    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton, "Please confirm you want to allow outside world to create invoices in your store (via API).");
-    ViewBag.MainTitle = "Pay Button";
+    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton);
+    ViewBag.Title = "Pay Button";
```
Comment (Kukks) [inline BTCPayServer/Views/Stores/PayButtonEnable.cshtml:4]:
```suggestion
```

File: BTCPayServer/Views/Stores/PayButtonEnable.cshtml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ﻿@{
     Layout = "../Shared/_NavLayout.cshtml";
-    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton, "Please confirm you want to allow outside world to create invoices in your store (via API).");
-    ViewBag.MainTitle = "Pay Button";
+    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton);
```
Comment (Kukks) [inline BTCPayServer/Views/Stores/PayButtonEnable.cshtml:3]:
```suggestion
    ViewData.SetActivePageAndTitle(StoreNavPages.PayButton, "Pay Button");
```

**Review by Kukks — CHANGES_REQUESTED** [review]

**Review by Kukks — APPROVED** [review]

**Pull request merged**